.github/workflows/build-macos.yml

```diff
@@ -26,15 +26,12 @@ jobs:
     - name: Configure
       working-directory: ${{runner.workspace}}/build
       run: |
-          export PKG_CONFIG_PATH="$(brew --prefix libarchive)/lib/pkgconfig"
-          cmake $GITHUB_WORKSPACE -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DQt5_DIR=$(brew --prefix qt5)/lib/cmake/Qt5
+          cmake $GITHUB_WORKSPACE -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DCMAKE_PREFIX_PATH="$(brew --prefix qt@5);$(brew --prefix libarchive)" -DMACOS_BUNDLE_LIBS=ON -DMACOS_BUILD_DMG=ON
```

Comment:
Could you disable bundling libraries in CI, otherwise it will only work on 10.15 and above.

Reply:
Problem is, if i do that the resulting binary will not actually run unless you have homebrew and all required libraries already installed on the system you want to run the build on.